tools/infrastructure/api_compare.py

```diff
@@ -0,0 +1,368 @@
+#!/usr/bin/env python
+
+import json
+import sys
+import argparse
+import xml.etree.ElementTree as ET
+from collections import namedtuple
+reload(sys)
+
+# Command line arguments parsing
+parser = argparse.ArgumentParser()
+parser.add_argument(
+    "-f1", "--mobile_api_file",
+    type=str, default="MOBILE_API.xml",
+    help='Enter mobile api file name! ("MOBILE_API.xml" by default)'
+)
+
+parser.add_argument(
+    "-f2", "--hmi_api_file",
```

Comment:
@AByzhynar, perhaps, rather than use abstract names like `-f1` and `-f2`, be batter to use something more closer to subject, like:
- `-m` 'n `-h`
- `-mob` 'n `-hmi`
- `-maf` 'n `-haf`
- `-mapi` 'n `-hapi`